app/views/menu/menu.html.erb

```diff
@@ -88,13 +88,46 @@
           </script>
           <% end %>
           <table class="table table-striped">
+          <tr>
+    <th>Dish Name</th>
+    <th>Dinner/Supper</th>
+    <% if current_user %> <%# for staff %>
+        <th>HC Prep</th>
+        <th>HC Leftover</th> 
+        <th>BB Prep</th>
+        <th>BB Leftover</th>
+        <th>Buckley Prep</th>
+        <th>Buckley Leftover</th>
+    <% end %>
```

Comment:
this is good, but what about the other fields in the production sheet? We also need to show Portion size, mech soft, pureed, and diet as well.

Reply:
I thought about it, but that part is more an attribute to a dish but not specific to a daily menu, so I thought it can go into the recipe of a dish.